README.md

```diff
@@ -40,48 +40,47 @@ The first step involves deciding which organizations a rule applies to. This is
 
 The second step involves selecting the parts of the ODM data model or entities that the rule applies to. The entities that can be selected are:
 
-- Data contained in a table
-- Data contained in column(s) of table(s)
+* Data contained in a table
+* Data contained in column(s) of table(s)
 
-This step uses two columns, `table` and `variable`. The `table` column can specify the name(s) of the table this rules applies to and the `variable` column can specify the name(s) of the columns this rule applies to.
+This step uses two columns, `tableName` and `variableName`. The `tableName` column can specify the name(s) of the table this rules applies to and the `variableName` column can specify the name(s) of the columns this rule applies to.
 
 Some examples are given below:
 
 1. Selecting the `type` column in the `Sample` table
 
-   | ruleId | table  | variable |
-   | ------ | ------ | -------- |
-   | 1      | Sample | type     |
+    | tableName | variableName |
```

Comment:
I'm agnostic about the name change, but we should still include a `ruleID` I think.